src/NServiceBus.Core.Tests/Serializers/XML/Issue_2796.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace NServiceBus.Core.Tests.Serializers.XML
+{
+    using System;
+    using System.IO;
+    using NServiceBus.Serializers.XML.Test;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class Issue_2796
```

Comment:
That's a good point. This issue (and subsequently, this PR) originated in a patch that was merged into older versions of NSB, and it was (probably easier) to keep it in separate files. I can move the test, it makes sense. Is there a value in storing the issue number as metadata somehow? Or is a commit message enough?